engine/schema/src/com/cloud/host/dao/HostDaoImpl.java

```diff
@@ -993,8 +991,8 @@ public boolean updateState(Status oldStatus, Event event, Status newStatus, Host
             }
         }
         if (event.equals(Event.ManagementServerDown)) {
-            Float pingTimeout = NumbersUtil.parseFloat(_configDao.getValue("ping.timeout"), 2.5f);
-            Integer pingInterval = NumbersUtil.parseInt(_configDao.getValue("ping.interval"), 60);
+            float pingTimeout = ManagementServiceConfiguration.PingTimeout.value();
+            int pingInterval = ManagementServiceConfiguration.PingInterval.value();
             ub.set(host, _pingTimeAttr, ((System.currentTimeMillis() >> 10) - (long)(pingTimeout * pingInterval)));
```

Comment:
The value `pingTimeout * pingInterval` is used again, this can be consumed from the class/interface of `ManagementServiceConfiguration` via a `getTimeout()` that is also used in `AgentManagerImpl.java`.